concent_api/common/decorators.py

```diff
@@ -298,3 +312,30 @@ def wrapper(*args, **kwargs):
             )
             raise
     return wrapper
+
+
+def get_response_from_view_with_secure_retry_if_database_locked(
```

Comment:
@cameel look at my last comment in https://github.com/golemfactory/concent/issues/412